doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -5755,6 +5754,10 @@ List a saved search by database id. ::
 
   GET http://$SERVER/api/admin/savedsearches/$id
 
+Delete a saved search by database id. The ``unlink=true`` query parameter unlink links (Linked dataset or Dataverse collection) related to the deleted saved search. This parameter should be well considered as you cannot know if the saved search created the links or if someone else did via other API. Also, it may be followed ``/makelinks/all`` depending on the need if other saved searches could recreate some deleted links or by reindexing some Dataverse or Dataset. ::
```

Comment:
```suggestion
Delete a saved search by database id.

The ``unlink=true`` query parameter unlinks all links (linked dataset or Dataverse collection) associated with the deleted saved search. Use of this parameter should be well considered as you cannot know if the links were created manually or by the saved search. After deleting a saved search with ``unlink=true`, we recommend running ``/makelinks/all`` just in case there was a dataset that was linked by another saved search. (Saved searched can link the same dataset.) Reindexing might be necessary as well. ::
```

Reply:
Committed with little modifications : moved `::` on first sentence + added one missing `